testing/embedding/android/focus_new_activity_back/.flutter-plugins

```diff
@@ -0,0 +1 @@
+android_intent=/Users/mattcarroll/.pub-cache/hosted/pub.dartlang.org/android_intent-0.3.1+1/
```

Comment:
Pretty sure this should be reverted. We probably should add files with this extension to the `.gitignore`.